openmc/stats/multivariate.py

```diff
@@ -613,6 +616,117 @@ def from_xml_element(cls, elem):
         origin = [float(x) for x in elem.get('origin').split()]
         return cls(r, phi, z, origin=origin)
 
+class MeshIndependent(Spatial):
+    """Spatial distribution for a mesh.
+
+    This distribution allows one to specify a mesh to sample over and the scheme to sample over the entire mesh. 
+
+    .. versionadded:: 0.13
+
+    Parameters
+    ----------
+    mesh : openmc.MeshBase
+        The mesh instance used for sampling, mesh is written into settings.xml, mesh.id is written into the source distribution
+    strengths : Iterable of Real, optional
+        A list of values which represent the weights of each element
+
+
+    Attributes
+    ----------
+    mesh : openmc.MeshBase
+        The mesh instance used for sampling, mesh is written into settings.xml, mesh.id is written into the source distribution
+    strengths : Iterable of Real, optional
+        A list of values which represent the weights of each element
+
+    """
+
+    def __init__(self, mesh, strengths=None, volume_normalized=False):
```

Comment:
I'd say the default here would be `True` in light of #2142. What do you think @eepeterson?

Reply:
I ended up changing the default value to True in the latest commit. I think it may be better to have the default be False so that first time users are not confused about why the relative strengths are different than the values passed in if they do not use/know about the volume normalized option

Reply:
I'd agree with @pshriwise that the default should be `True`. I think it makes sense for the default UM sampling to be uniform by volume not uniform by index.